### DISCUSSION TITLE
win端打不开面板

### DISCUSSION THREAD

**Discussion by rainbow175655** [clash-verge-rev/clash-verge-rev#1752] [Q&A] [answered]
win端打不开面板

**Comment by wonfen** [comment]
下载2.0版试试。

**Reply by rainbow175655** [reply]
刚下完2.0  还是只能在后台，面板打不开
![屏幕截图 2024-11-24 114611](https://github.com/user-attachments/assets/74de4441-dd57-4f11-b300-445a8464ff49)

**Reply by rainbow175655** [reply]
https://github.com/user-attachments/assets/b8d10c4b-79dc-4940-9118-c4aa0cef2656

**Reply by rainbow175655** [reply, marked as answer]
下载了176M的完整安装包就好了

**Reply by wonfen** [reply]
重装webview2

**Reply by lff231118** [reply]
> 重装webview2
同样问题，
重装完整版也不行。之前2.0也一样！2.0之前的没有这个问题！
